docs/_includes/usage/cf-layout/usage.md

```diff
@@ -1290,208 +1290,181 @@ height of the next parent using `position: relative;`. This means that the
 
 
 ## Featured content module
-{% comment %}
-TODO: Revist the naming of these three modules. The layouts are more about
-positioning the image and less about the type of content they host.
-{% endcomment %}
 
-Featured content module, like a hero, consists of headline/text/optional call
-to action along with a visual. It is intended to be used in a main content
-column next to a sidebar.
+Featured content modules, like a hero, consist of headline and description,
+an optional call to action, and a static or video visual.
+They are intended to be used in a main content column next to a sidebar.
 
 Text is full width & displayed above the visual in the default/mobile view.
-At larger screen sizes, the text occupies a fixed portion of the screen (equal
-to the width of 5 of 12 columns at `701px` & 3 of 12 columns at `901px` for
-desktop). The visual occupies the remaining space. The visual should be
-`640x360` (16x9 ratio) and resize to fit the height of the Featured Content
-Module with a static width and is anchored left when it becomes too wide for
-the available space.
-
-<div class="block block__border block__flush-sides">
-    <section class="o-featured-content-module">
-        <div class="o-featured-content-module_text">
-            <div class="a-heading a-heading__icon">
-                {% include icons/speech-bubble.svg %}
-                Featured
-            </div>
-            <h2>Feature title</h2>
-            <p>
-                Lorem ipsum dolor sit amet, ei ius adhuc inani iudico, labitur
-                instructior ex pri. Cu pri inani constituto, cum aeque noster
-                commodo cu.
-            </p>
-            <a class="a-link
-                      a-link__jump
-                      a-link__icon-after-text">
-                <span class="a-link_text">Read more about the feature</span>
-                {% include icons/right.svg %}
-            </a>
-        </div>
-        <div class="o-featured-content-module_visual">
-            <img class="o-featured-content-module_img"
-                 src="http://placekitten.com/g/540/270">
-        </div>
-    </section>
-</div>
+At larger screen sizes, the image moves to the right of the text,
+occupying a fixed width of 270px (equal to 3 of 12 columns at max page width).
+When creating a static image for the visual, it should be 1076px × 606px
+(a 16:9 aspect ratio), which is 2x the rendered with at the maximum size
+at which the visual will be seen (on a 600px wide display).
+
+By default, the image is anchored on its left when the FCM is viewed on a
```

Comment:
Should it say `anchored on its center-left` or `anchored on its middle-left`, as that's what it looks like it is doing?

Reply:
I tried to clarify this below in the right-anchored description (and I should have written it the same way here), but it looks like that was confusing to you also, so perhaps I should go back to the drawing board 😆 

How does this sound (including some minor edits to the previous paragraph)?

---

Text is full width and displayed above the visual in the default/mobile view.
When creating a static image for the visual, it should be 1076px × 606px
(a 16:9 aspect ratio), which is 2x the rendered width at the maximum size
at which the visual will be seen (on a 600px wide display).

At larger screen sizes, the image moves to the right of the text,
occupying a fixed width of 270px (equal to 3 of 12 columns at max page width).
By default, the left edge of the image is anchored to the left side of the
visual's 270px-wide area, and the right side is cropped off.
The height of the visual area decreases as screen size increases,
resulting in slightly different image cropping at different screen sizes,
but the left edge of the image remains anchored in view.
See below for modifiers that change the image anchoring.

Reply:
Updated in 43458fae.

Reply:
> The height of the visual area decreases as screen size increases,
resulting in slightly different image cropping at different screen sizes,
but the left edge of the image remains anchored in view.

Still not totally clear how the cropping works without looking at it closer. The top AND bottom get cropped as screen size increases? Or is it anchored to upper-left and the bottom only gets cropped?

Reply:
Also, should it be `Text is full-width …`? Not sure what we use elsewhere.

Reply:
> Still not totally clear how the cropping works without looking at it closer. The top AND bottom get cropped as screen size increases? Or is it anchored to upper-left and the bottom only gets cropped?

The full height of the image remains visible; top and bottom are not cropped at all. I can add that in.

This is extraordinarily difficult to describe in words. The best thing to do is open the responsive design mode and play around with it. Maybe I should literally say that in the docs, too :)

> Also, should it be `Text is full-width …`? Not sure what we use elsewhere.

This phrase is leftover from the previous iteration of the docs. It is probably assumed to be true when one is talking about the mobile view, so it may not be necessary to state it that way. How about... "The text and visual are stacked vertically at screen widths of 600px and under."?